third_party/icu/source/i18n/decNumber.cpp

```diff
@@ -178,16 +178,13 @@
 
 #if defined(STARBOARD)
 #include "starboard/client_porting/poem/assert_poem.h"
-#include "starboard/client_porting/poem/string_poem.h"
 #endif  // defined(STARBOARD)
 
 #include <stdlib.h>                /* for malloc, free, etc.  */
 
```

Comment:
Maybe remove this empty line we've inserted.
https://chromium.googlesource.com/chromium/deps/icu/+/d1e2bd28/source/i18n/decNumber.cpp#179